configuration/roles.d/appkernels.json

```diff
@@ -0,0 +1,23 @@
+{
+  "roles": {
+    "default": {
+      "permitted_modules": [
+        {
+          "name": "app_kernels",
+          "title": "App Kernels",
+          "position": 400,
+          "permitted_modules": [
+            "app_kernel_viewer",
+            "app_kernel_explorer",
+            "app_kernel_notification",
+            "app_kernel_performance_map"
+          ],
+          "javascriptClass": "XDMoD.Module.AppKernels",
+          "javascriptReference": "CCR.xdmod.ui.appKernels",
+          "userManualSectionName": "App Kernels",
+          "tooltip": "Displays data referencing the reliability and performance of grid resources"
+        }
+      ]
+    }
+  }
+}
```

Comment:
Why did the file contents change?  The indentation and some of the ordering is different as well as the removal of the + prefixes.

Reply:
- I'm honestly not sure why the `tooltip` / `userManualSectionName` properties were flipped, I can certainly put those back. 
- The `+` was removed as it doesn't do anything with the current configuration code. 
- And the spacing change was just flub on my part, I must have formatted it before saving. I can also change this back to 4 space min.

Reply:
I just saw the `referencing` / `reflecting` change, not sure what was up with that but I've reverted it.